ERROR: ghc-cabal: /Users/lukexi/Code/ghc-ios-scripts/i386-apple-darwin11-ranlib: does not exist

This is what I get; my ghc-ios-scripts clone is at b4a8022; I'm on OS X 10.9.5 with GHC 7.8.3 from HT:

```
~$ installGHCiOS.sh 
Downloading LLVM 3.0...
...
Downloading GHC for iOS devices...
...
...
for i in  rts/dist/build/libHSrts.a rts/dist/build/libHSrts_p.a rts/dist/build/libHSrts_l.a rts/dist/build/libHSrts_debug.a rts/dist/build/libHSrts_thr.a rts/dist/build/libHSrts_thr_debug.a rts/dist/build/libHSrts_thr_l.a rts/dist/build/libHSrts_thr_p.a  rts/dist/build/libCffi.a rts/dist/build/libCffi_p.a rts/dist/build/libCffi_l.a rts/dist/build/libCffi_debug.a rts/dist/build/libCffi_thr.a rts/dist/build/libCffi_thr_debug.a rts/dist/build/libCffi_thr_l.a rts/dist/build/libCffi_thr_p.a; do case $i in *.a) /usr/bin/install -c -m 644  $i  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0"; true  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0"/`basename $i` ;; *.dll) /usr/bin/install -c -m 755  $i  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0" ; :  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0"/`basename $i` ;; *.so) /usr/bin/install -c -m 755  $i  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0" ;; *.dylib) /usr/bin/install -c -m 755  $i  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0";; *) /usr/bin/install -c -m 644  $i  "/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/rts-1.0"; esac; done
"inplace/bin/ghc-cabal" copy libraries/ghc-prim dist-install ":" '' '/usr/local' '/usr/local/lib/i386-apple-darwin11-ghc-7.8.3' '/usr/local/share/doc/ghc/html/libraries' 'v p'  
Installing library in
/usr/local/lib/i386-apple-darwin11-ghc-7.8.3/ghc-prim-0.3.1.0
ghc-cabal: /Users/lukexi/Code/ghc-ios-scripts/i386-apple-darwin11-ranlib: does
not exist
make[1]: *** [install_packages] Error 1
make: *** [install] Error 2
```

Also, this might be relevant:

```
x ghc-7.8.3/mk/project.mk
x ghc-7.8.3/mk/install.mk.in
x ghc-7.8.3/bindist.mk
x ghc-7.8.3/libraries/gen_contents_index
x ghc-7.8.3/libraries/prologue.txt
sed: RE error: illegal byte sequence    <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
checking for path to top of build tree... /private/tmp/ghc-7.8.3-i386
Build platform inferred as: i386-apple-ios
Host platform inferred as: i386-apple-ios
Target platform inferred as: i386-apple-ios
```

I found this line in the installation script:

```
find . -type f -not -name .DS_Store -not -name "*.a" -print0 | xargs -0 sed -i '' 's|/Users/lukexi/Code/ghc-ios-scripts/||g'
```

— copy and pasting it into Terminal.app yields:

```
~$ find . -type f -not -name .DS_Store -not -name "*.a" -print0 | xargs -0 sed -i '' 's|/Users/lukexi/Code/ghc-ios-scripts/||g'
sed: RE error: illegal byte sequence
```

--- a/installGHCiOS.sh
+++ b/installGHCiOS.sh
@@ -21,6 +21,7 @@ cd ghc-7.8.3-arm
 # Remove befuddling inclusion of my local paths
 LC_CTYPE=C 
 LANG=C
+LC_ALL=C
 find . -type f -not -name .DS_Store -not -name "*.a" -print0 | xargs -0 sed -i '' 's|/Users/lukexi/Code/ghc-ios-scripts/||g'
 
 ./configure
@@ -43,6 +44,7 @@ cd ghc-7.8.3-i386
 # ditto above
 LC_CTYPE=C 
 LANG=C
+LC_ALL=C
 find . -type f -not -name .DS_Store -not -name "*.a" -print0 | xargs -0 sed -i '' 's|/Users/lukexi/Code/ghc-ios-scripts/||g'
 
 ./configure

